charts/internal/shoot-crds/charts/volumesnapshots/templates/crd-volumesnapshotclasses.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: volumesnapshotclasses.snapshot.storage.k8s.io
+  annotations:
+    resources.gardener.cloud/keep-object: "true"
+spec:
```

Comment:
I noticed that `resources.gardener.cloud/keep-object` is set to `true` for `volumesnapshotclasses`, but not for the other 2 CRDs. Is there a special reason for this? I understand that it's "safe" to keep the volumesnapshotclasses (and hence the CRDs) as there are no provider resources backing them, but would it actually hurt if we deleted them? Is it perhaps to avoid another "race condition", e.g. `volumesnapshots` could not be deleted if their `volumesnapshotclass` was deleted?

Reply:
> Is it perhaps to avoid another "race condition", e.g. volumesnapshots could not be deleted if their volumesnapshotclass was deleted?

Yep. Explained better in https://github.com/gardener/gardener-extension-provider-azure/pull/302.